src/core/game.ts

```diff
@@ -96,25 +96,44 @@ export class Game {
       Object.assign(list, (game.entities as any)[key]);
       return list;
     }, {});
+    let { walls, players }  = game.entities;
 
     events.forEach(event => {
       let sender = allEntities[event.initiator];
       let receiver = allEntities[event.receptor];
       switch(event.type) {
-        case 'COLLISION':
+        case 'COLLISION': // handles static motion
           switch(sender.type) {
             case 'player': Player.collideWith(sender as PlayerState, receiver, game); break;
             case 'bullet': Bullet.collideWith(sender as BulletState, receiver, game); break;
           }
           break;
-        case 'SPAWN_BULLET':
+        case 'SPAWN_BULLET': // spawns the missiles
```

Comment:
Its clear enough for the other ones as well.  I'd say "Collision" and "Movement" is even more clear than "Static motion" and "Dynamic motion", I'm not even sure what "Static motion" means.